docs/src/modules/components/MarkdownElement.js

```diff
@@ -412,6 +416,9 @@ const Root = styled('div')(
       '&:hover': {
         textDecorationColor: 'inherit',
       },
+      '&:focus-visible': {
+        outline: `3px solid ${alpha(lightTheme.palette.primary[500], 0.5)}`,
```

Comment:
Is their a specific reason to declare that multiple times?

Could the selector `& :focus-visible` (with a space between `&` and `:` at the root of the theme do the job, or does it conflict with other CSS rules?

Reply:
It's because I'm adding it to different elements — I can try doing it globally and see how it works!

Reply:
Iterated a little bit on this one — lmk what you think!